CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ### To be Released
+* feat(dashboard): add command `dashboard` to open dashboard of specified app on default browser [#606](https://github.com/Scalingo/cli/pull/686)
```

Comment:
typo in issue number + nitpick:
```suggestion

* feat(dashboard): add command `dashboard` to open dashboard of specified app on default browser [#686](https://github.com/Scalingo/cli/pull/686)
```

Reply:
Does the number between brackets refer to the issue number or the PR number ?

Reply:
When you write something inside square brackets followed by brackets, it's the markdown syntax for links (https://www.markdownguide.org/basic-syntax#links). In this case, we want to link with the PRs. Hence what is inside the square brackets should be the PR number, and the URL in brackets should target the PR

Reply:
My brain is so slow sometimes. Thanks I'm on it.